quipucords/api/source/model.py

```diff
@@ -55,35 +56,16 @@ class Source(models.Model):
     options = models.ForeignKey(
         SourceOptions, null=True, on_delete=models.CASCADE)
     credentials = models.ManyToManyField(Credential)
-    # Source also has the field hosts, which is created by the
-    # ForeignKey in HostRange below.
+    hosts = models.TextField(unique=False, null=False)
 
     def __str__(self):
         """Convert to string."""
-        return '{ id:%s, name:%s, type:%s }' %\
-            (self.id, self.name, self.source_type)
+        return '{ id:%s, name:%s, type:%s , hosts:%s}' %\
+            (self.id, self.name, self.source_type, self.hosts)
```

Comment:
I feel like this could potentially be a huge string to print out. What if there are 10K hosts? Might be best to leave it out.

Reply:
Removed.